fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4311,13 +4311,14 @@ class FileBackupAgentImpl {
 			    .detail("OverrideTargetVersion", targetVersion);
 		}
 
-		Optional<RestorableFileSet> restoreSet = wait(bc->getRestoreSet(targetVersion));
-
-		if (!restoreSet.present()) {
-			TraceEvent(SevWarn, "FileBackupAgentRestoreNotPossible")
-			    .detail("BackupContainer", bc->getURL())
-			    .detail("TargetVersion", targetVersion);
-			throw restore_invalid_version();
+		if (!CLIENT_KNOBS->IGNORE_LOG_FILES) {
+			Optional<RestorableFileSet> restoreSet = wait(bc->getRestoreSet(targetVersion));
+			if (!restoreSet.present()) {
+				TraceEvent(SevWarn, "FileBackupAgentRestoreNotPossible")
+					.detail("BackupContainer", bc->getURL())
+					.detail("TargetVersion", targetVersion);
+				throw restore_invalid_version();
+			}
```

Comment:
Please revert this change. This is for parallel restore, not the regular restore.

Instead, the knob should be used at `StartFullRestoreTaskFunc::_execute()` to completely ignore log files:
```
		for (const LogFile& f : restorable.get().logs) {
			files.push_back({ f.beginVersion, f.fileName, false, f.blockSize, f.fileSize, f.endVersion });
		}
```

Reply:
Thanks, done.